about.html

```diff
@@ -0,0 +1,165 @@
+                  <li><a href="#">Contact</a></li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+      </nav>
+      <nav class="desktop-only mobile-none">
+        <div class="top-bar">
+          <ul class="sm-menu">
+            <li class="small-menu"><img src="assets/images/facebook.png" alt="logo facebook"></li>
+            <li class="small-menu"><img src="assets/images/twitter.png" alt="logo twitter"></li>
+            <li class="small-menu">English</li>
+            <li class="small-menu">My Page</li>
+            <li class="small-menu">Logout</li>
+          </ul>
+        </div>
+        <div class="bottom-bar">
+          <img src="assets/images/advanced-energy-vector-logo.png" alt="logo" />
+          <ul class="big-menu">
+            <!-- </ul> -->
+            <!-- <ul class="big-menu-right"> -->
+              <li class="big-menu-right">
+                <a href="index.html">Index</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="about.html">About</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="#program">Program</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="#speak">Speakers</a>
+              </li>
+              <li class="big-menu-right">
+                <a href="#sponsors">Sponsors</a>
+              </li>
+              <li class="big-menu-right"><button class="contact">Contact Us</button>
+            </li>
+          </ul>
+        </div>
+      </nav>
+      <!-- Top Menu Desktop-->
+    </header>
+  </div>
+  <section>
+    <br>
+    <h1 class="event-name">Energy Conference<br>
+      Goiânia, Brazil</h1>
+    <p class="event-description mobile-none">
+      Lorem ipsum dolor sit amet consectetur adipisicing elit. Natus, minima saepe incidunt quis ex eum voluptatem rem
+      unde fuga repudiandae iure? Animi porro laboriosam odio dolore quisquam nam veritatis iste.<br><br> Lorem ipsum
+      dolor sit amet consectetur adipisicing elit. Non deleniti laboriosam quas, consequuntur sint hic adipisci
+      perferendis nisi corrupti odit dolores similique fugit minus libero optio quos rerum suscipit quae.<br><br>
+      Similique corporis in illo delectus! Commodi ducimus cum quam expedita, qui sit nisi itaque, asperiores officiis
+      facilis in maxime nesciunt voluptatibus. Quaerat debitis ea quidem? Minus, cupiditate? Soluta, eaque
+      dolorum.<br><br>
+      Ea quos labore officiis explicabo possimus? Suscipit beatae aliquid ipsam ab nisi perferendis aut magni! A quos
+      voluptate, cum eius, dicta quisquam sapiente dolor ea possimus, dignissimos quod delectus ipsa.<br><br>
+      Fugiat facere, dolorum cumque provident nisi dolore. Quo tempora suscipit accusantium perspiciatis quos aliquid
+      tenetur inventore ullam ut quae? Vel optio inventore dignissimos repellat unde magni beatae! Molestiae, optio ab?
+    </p>
```

Comment:
- [ ] __[OPTIONAL]__ The text in this section is more than what's in the design and this section should also be centred, not left-aligned.

Yours | Original
---- | ----
![image](https://user-images.githubusercontent.com/16058470/143622849-680be87b-7ae3-47cf-831c-2a66c7eb472c.png) | ![image](https://user-images.githubusercontent.com/16058470/143622805-6754cda9-9655-4773-89cf-b3e9a6fd2e89.png)